util/src/main/scala/geotrellis/util/np/package.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.util
+
+import spire.syntax.cfor.cfor
+
+package object np {
+
+  /**
+    * Return evenly spaced numbers over the specified interval
+    * @param min Min bound of the interval, inclusive
+    * @param max Max bound of the interval, inclusive
+    * @param points Number of samples to generate
+    * @return
+    */
+  def linspace(min: Double, max: Double, points: Int): Array[Double] = {
+    val d = new Array[Double](points)
+    cfor(0)(_ < points, _ + 1) { i =>
+      d(i) = min + i * (max - min) / (points - 1)
+    }
+    d
+  }
+
+  /**
+    * Compute percentile at the given breaks using the same algorithm as numpy
+    *
+    * https://docs.scipy.org/doc/numpy/reference/generated/numpy.percentile.html
+    * https://en.wikipedia.org/wiki/Percentile
+    *
+    * @param data
+    * @param pctBreaks
+    * @return
+    */
+  def percentile(data: Array[Double],
+                 pctBreaks: Array[Double]): Array[Double] = {
+    // TODO: Include the isData filter check in the Tile implicit method
+    // val arr = data.filter(isData(_))
```

Comment:
Should these commented out lines be removed? Mb it makes sense to add into the comments that if it is a tile - user probably may want to filter noDataValues?

Reply:
Oooops yeah I'll delete. Presumably for the generic version of this method, the user would just filter the Array[Double] before passing to this method. 

I kept the nodata filter in the Tile version since that's what we had. Should we remove the filter there as well? I'm waffling on it.

Reply:
nodata filter should be present in a Tile version, that's all good :+1: 
The comment was only about adding comments for this function and removing commented out code : D

Reply:
Ah sure thing. Updated with comments removed.